checkstyle/suppressions.xml

```diff
@@ -111,17 +114,8 @@
               files="Values.java"/>
 
     <suppress checks="NPathComplexity"
-              files="ConnectRecord.java"/>
-    <suppress checks="NPathComplexity"
-              files="ConnectSchema.java"/>
-    <suppress checks="NPathComplexity"
-              files="FileStreamSourceTask.java"/>
-    <suppress checks="NPathComplexity"
-              files="JsonConverter.java"/>
-    <suppress checks="NPathComplexity"
-              files="DistributedHerder.java"/>
-    <suppress checks="NPathComplexity"
-              files="ConnectHeaders.java"/>
+              files="(AbstractStatus|ConnectHeaders|ConnectRecord|ConnectSchema|DistributedHerder|FileStreamSourceTask|JsonConverter|KafkaConfigBackingStore).java"/>
```

Comment:
cc @rhauch

Reply:
@ijuma I presume we don't want to correct these NPath exceptions with this PR, so these are acceptable for now. I can log a separate issue for reducing the exceptions to this rule, since we may want to backport some of them to make cross-branch maintenance easier.

Reply:
See #5051 and KAFKA-6926.

Reply:
Perfect. Yes, I didn't want to tackle it in this PR, but wanted you to be aware of it so that you can decide the best course of action.